sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -65,7 +66,7 @@
 
 
 @pytest.mark.parametrize(
-    "Estimator, method, data",
+    "estimator, method, data",
```

Comment:
Can you revert the change in this test. this is unrelated and I think it is fine to have `Estimator` because it represent a class and not an instance.